packages/mongo-nav/src/helpers/ProjectStatusBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Badge from '@leafygreen-ui/badge';
+import { css } from '@leafygreen-ui/emotion';
+import { ProjectStatus } from '../types';
+
+const projectStatusBadgeMargin = css`
+  margin-right: 20px;
+`;
+
+const Colors = {
```

Comment:
It may be worth exporting this enum from badge!

Reply:
good call honestly i'll do that now

Reply:
it already was 🤦‍♀, but using it now